docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1704,11 +1707,14 @@ you must configure the annotation processor in your build tool:
         <annotationProcessorPaths>
             <path>
                 <groupId>org.hibernate.orm</groupId>
-                <artifactId>hibernate-jpamodelgen</artifactId>
+                <artifactId>hibernate-processor</artifactId>
                 <!-- Note, no artifact version is required, it's managed by Quarkus.  -->
             </path>
             <!-- other processors that may be required by your app -->
         </annotationProcessorPaths>
+        <annotationProcessors>
+            <annotationProcessor>org.hibernate.processor.HibernateProcessor</annotationProcessor>
```

Comment:
Why do we need this now?

Reply:
oh that's on me 🙈, previous processor `org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor` is deprecated in favour of this "new" one `org.hibernate.processor.HibernateProcessor` (but essentially it is just a package/class name change as `JPAMetaModelEntityProcessor extends HibernateProcessor`) and I missed that only the new one is referenced in the `meta-inf/services` so while being explicit about which processor to apply does not hurt, it is also redundant here 🙈

Reply:
Removed